R/prepare_ictr_company.R

```diff
@@ -11,17 +14,9 @@
 #' @return A dataframe that prepares the final output of ictr_company
 #'
 #' @export
-#'
+#' @keywords internal
```

Comment:
This makes the old function dissapear from the website's Reference (and R's index) -- but the helpfile still exists at the old addresses (online and locally).